### PR TITLE
Clarify methods

### PR DESCRIPTION
**Pull request by tej99**
# Motivation and Context
While trying to understand the ActionPlanJobServiceImpl some refactoring was done.

# What has changed
ActionPlanJobServiceImpl.java, no functional changes.

# How to test?
mvn test

# Links
https://trello.com/c/Eknzy8H2/119-tbl123-link-action-rules-to-action-plans-at-create-time-create

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java [hunk 1]
```diff
@@ -47,14 +47,11 @@
...
     final DistributionInfo distInfo = new DistributionInfo();
-
     final List<ActionType> actionTypes = actionTypeRepo.findAll();
 
-    if (!CollectionUtils.isEmpty(actionTypes)) {
-      for (final ActionType actionType : actionTypes) {
-        final List<InstructionCount> instructionCounts = processActionType(actionType);
-        distInfo.getInstructionCounts().addAll(instructionCounts);
-      }
+    for (final ActionType actionType : actionTypes) {
```
Comment (benjefferies) [inline src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java]: 👍

File: src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java [hunk 2]
```diff
@@ -117,6 +117,18 @@ public void testFailToGetAnyAction() throws Exception {
     verify(actionProcessingService, times(0)).processActionCancel(any(Action.class));
   }
 
+  /**
+   * We retrieve actionTypes but then exception thrown when retrieving actions.
+   *
+   * @throws Exception oops
```
Comment (benjefferies) [inline src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java]: If the test method was `testExceptionNotThrowWhenNoActionTypesLeads` this comment would probably be redundant. Also `@throws Exception oops` is pointless :)

Comment (tej99) [reply src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java]: copy/paste problems... cheers 👍

**Comment by PurpleBooth** [comment]
[Tidy ActionService Action Plan Job Service createAndExecuteActionPlanJob](https://trello.com/c/ckOiSs08/131-tidy-actionservice-action-plan-job-service-createandexecuteactionplanjob)

**Comment by PurpleBooth** [comment]
[Tidy ActionService Action Plan Job Service createAndExecuteActionPlanJob](https://trello.com/c/ckOiSs08/131-tidy-actionservice-action-plan-job-service-createandexecuteactionplanjob)

**Pull request merged**